src/service/print/dataImporter.js

```diff
@@ -1,16 +1,18 @@
-const fetch = require("node-fetch");
+
 const envVariables = require("../../envVariables");
+const fetch = require("node-fetch");
 
-class PDFDataImportError {
+
+class PrintDocxDataImportError {
   constructor(message) {
     this.message = message;
-    this.name = "PDFDataImportError";
+    this.name = "PrintDocxDataImportError";
   }
 }
 
-function getItemsFromItemset(itemsetID,marks) {
+function getItemsFromItemset(itemsetID, marks) {
   let status;
-  const urlItemset = `${envVariables.baseURL}/action/itemset/v3/read/${itemsetID}`;
+  const urlItemset = `${envVariables.LEARNER_SERVICE_URL}itemset/v3/read/${itemsetID}`;
```

Comment:
Why are you changing `baseURL` to `LEARNER_SERVICE_URL`

Reply:
As mentioned in the yml file  https://github.com/project-sunbird/sunbird-devops/blob/release-4.6.0-vdn/ansible/roles/kong-api/defaults/main.yml  
Item set read URL will point to this - `upstream_url: "{{ learning_service_url }}/itemset/v3/read"`  , and even I changed the envVariables.js file also.